liberty-maven-plugin/src/main/resources/io/openliberty/tools/maven/MvnMessages.properties

```diff
@@ -356,21 +356,21 @@ error.uninstall.feature.fail.useraction=Check the return code and execution fail
 
 
 #need to define new message string
-info.toolchain.initialized=CWWKM4100I: Using toolchain from build context: {0}
-info.toolchain.initialized.explanation=Toolchain is configured for liberty server tasks from <jdkToolchain> and user's .m2/toolchain.xml.
+info.toolchain.initialized=CWWKM4100I: Using toolchain from build context: {0}.
+info.toolchain.initialized.explanation=Toolchain is configured for Liberty server tasks from <jdkToolchain> and user's .m2/toolchain.xml.
 info.toolchain.initialized.useraction=No action required.
 
 #need to define new message string
-warn.toolchain.not.available=CWWKM4100W: Toolchain requested but not available for requirement : {0}. Toolchain will not be honored
-warn.toolchain.not.available.explanation=Toolchain is configured for liberty server tasks in <jdkToolchain> but jdkHome is not configured in user's .m2/toolchain.xml.
+warn.toolchain.not.available=CWWKM4100W: Toolchain requested but not available for JDK requirement : {0}. Toolchain will not be honored.
```

Comment:
You still did not put the reason in the actual warning message. As far as I know, the end user has no way of seeing the explanation that you provided in the next part of this message. Liberty posts their messages in documentation somewhere with those details I think, but we do not. An end user would have to dive into our source repo and find this message file to see it.